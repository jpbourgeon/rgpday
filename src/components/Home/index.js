import React from 'react'
import PropTypes from 'prop-types'
import { withStyles } from '@material-ui/core/styles'
import Paper from '@material-ui/core/Paper'
import Typography from '@material-ui/core/Typography'
import Grid from '@material-ui/core/Grid'
import CallToAction from '../CallToAction'
import Tile from '../Tile'
import BookStore from './images/bookstore.jpg'
import Coconstruction from './images/coconstruction.jpg'
import Conference from './images/conference.jpg'
import GameBoard from './images/gameboard.jpg'

const styles = theme => {
  return {
    layout: {
      width: 'auto',
      marginLeft: theme.spacing.unit * 3,
      marginRight: theme.spacing.unit * 3,
      [theme.breakpoints.up(1100 + theme.spacing.unit * 3 * 2)]: {
        width: 1100,
        marginLeft: 'auto',
        marginRight: 'auto'
      }
    },
    mainFeaturedPost: {
      background: `url('${BookStore}') no-repeat`,
      backgroundSize: `100% 100%`,
      color: theme.palette.common.white,
      marginBottom: theme.spacing.unit * 2
    },
    mainFeaturedPostContent: {
      padding: `${theme.spacing.unit * 6}px`,
      paddingBottom: `${theme.spacing.unit * 2}px`,
      [theme.breakpoints.up('md')]: {
        paddingRight: 0
      }
    },
    mainGrid: {
      marginTop: theme.spacing.unit * 3
    },
    avatar: {
      margin: 10,
      width: 100,
      height: 100,
      float: 'left'
    }
  }
}

const HomeComponent = props => {
  const { classes } = props
  return (
    <div className={classes.layout}>
      <main>
        <Paper className={classes.mainFeaturedPost}>
          <Grid container alignItems='center'>
            <Grid item xs={12} md={6}>
              <div className={classes.mainFeaturedPostContent}>
                <Typography variant='h4' color='inherit' gutterBottom>
                    Une journée pour expérimenter le RGPD et en comprendre les enjeux
                </Typography>
                <Typography variant='h5' color='inherit' paragraph>
                  La règlementation sur les données à caractère personnel est une opportunité
                  pour mieux gérer et pour valoriser les données de l'entreprise.
                  <br />Cette journée d'échange est l'occasion de le découvrir...
                </Typography>
              </div>
            </Grid>
            <Grid item xs={12} md={6}>
              <CallToAction xs={10} md={12} />
            </Grid>
          </Grid>
        </Paper>

        <Grid container spacing={40}>

          <Tile
            title='Une conférence'
            description={`Quel bilan pour le RGPD depuis son entrée en vigueur ? Comment l'insérer dans les outils de gestion des documents et des données au sein de l'entreprise ? Cette conférence fait le point sur la mise en oeuvre du RGPD au quotidien.`}
            image={Conference}
          />

          <Tile
            title='Un serious game'
            description={`Pour mettre en pratique de façon ludique les connaissances théoriques et défier les équipes des autres participants !`}
            image={GameBoard}
          />

          <Tile
            title='Une séance de co-constructrion'
            description={`Pour mettre en commun les enseignements de la journée et dégager une vision sur les actions à entreprendre pour tendre vers la conformité.`}
            image={Coconstruction}
          />

        </Grid>

      </main>
    </div>
  )
}

HomeComponent.propTypes = {
  classes: PropTypes.object.isRequired
}

export default withStyles(styles)(HomeComponent)
