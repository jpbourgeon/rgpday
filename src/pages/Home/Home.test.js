/* global describe, it, expect */
import React from 'react'
import renderer from 'react-test-renderer'
import Home from './'
import jssSerializer from 'src/utils/jssSnapshotSerializer'

expect.addSnapshotSerializer(jssSerializer)

describe('The Home component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Home />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
