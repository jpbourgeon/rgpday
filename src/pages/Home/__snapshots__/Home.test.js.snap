// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Home component should render correctly 1`] = `
<div
  className="layout-1"
>
  <main>
    <div
      className="root-6 elevation2-10 rounded-7 mainFeaturedPost-2"
    >
      <div
        className="container-33 center-41"
      >
        <div
          className="item-34 12-73 6-95"
        >
          <div
            className="mainFeaturedPostContent-3"
          >
            <h4
              className="root-130 h4-145 colorInherit-159 gutterBottom-157"
            >
              Une journée pour expérimenter le RGPD et en comprendre les enjeux
            </h4>
            <p
              className="root-130 h5-146 colorInherit-159 paragraph-158"
            >
              La règlementation sur les données à caractère personnel est une opportunité pour mieux gérer et pour valoriser les données de l'entreprise.
              <br />
              Cette journée d'échange est l'occasion de le découvrir...
            </p>
          </div>
        </div>
        <div
          className="item-34 12-73 6-95"
        >
          <div
            className="container-33 40-59 column-36 center-41"
          >
            <div
              className="item-34 10-71 12-87 12-101"
            >
              <div
                className="root-6 elevation1-9 rounded-7 cta-166"
              >
                <h6
                  className="root-130 h6-147"
                >
                  <a
                    className="root-130 colorSecondary-161 inline-165 root-175 underlineHover-177"
                    href="/contact"
                    onClick={[Function]}
                  >
                    Contactez-moi !
                  </a>
                  <br />
                  pour organiser le RGPDay dans votre établissement
                </h6>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="container-33 40-59"
    >
      <div
        className="item-34 12-73 6-95"
      >
        <div
          className="root-6 elevation1-9 rounded-7 root-171 card-168"
        >
          <div
            className="cardDetails-169"
          >
            <div
              className="root-172"
            >
              <h5
                className="root-130 h5-146"
              >
                Une conférence
              </h5>
              <p
                className="root-130 subtitle1-148 paragraph-158"
              >
                Quel bilan pour le RGPD depuis son entrée en vigueur ? Comment l'insérer dans les outils de gestion des documents et des données au sein de l'entreprise ? Cette conférence fait le point sur la mise en oeuvre du RGPD au quotidien.
              </p>
            </div>
          </div>
          <div
            className="root-173 cardMedia-170"
            style={
              Object {
                "backgroundImage": "url(\\"conference.jpg\\")",
              }
            }
            title="Une conférence"
          />
        </div>
      </div>
      <div
        className="item-34 12-73 6-95"
      >
        <div
          className="root-6 elevation1-9 rounded-7 root-171 card-168"
        >
          <div
            className="cardDetails-169"
          >
            <div
              className="root-172"
            >
              <h5
                className="root-130 h5-146"
              >
                Un serious game
              </h5>
              <p
                className="root-130 subtitle1-148 paragraph-158"
              >
                Pour mettre en pratique de façon ludique les connaissances théoriques et défier les équipes des autres participants !
              </p>
            </div>
          </div>
          <div
            className="root-173 cardMedia-170"
            style={
              Object {
                "backgroundImage": "url(\\"gameboard.jpg\\")",
              }
            }
            title="Un serious game"
          />
        </div>
      </div>
      <div
        className="item-34 12-73 6-95"
      >
        <div
          className="root-6 elevation1-9 rounded-7 root-171 card-168"
        >
          <div
            className="cardDetails-169"
          >
            <div
              className="root-172"
            >
              <h5
                className="root-130 h5-146"
              >
                Une séance de co-constructrion
              </h5>
              <p
                className="root-130 subtitle1-148 paragraph-158"
              >
                Pour mettre en commun les enseignements de la journée et dégager une vision sur les actions à entreprendre pour tendre vers la conformité.
              </p>
            </div>
          </div>
          <div
            className="root-173 cardMedia-170"
            style={
              Object {
                "backgroundImage": "url(\\"coconstruction.jpg\\")",
              }
            }
            title="Une séance de co-constructrion"
          />
        </div>
      </div>
    </div>
  </main>
</div>
`;
